listenbrainz/spark/request_queries.json

```diff
@@ -1,57 +1,52 @@
 {
```

Comment:
Why are there fewer handlers now? I'm generally confused by this commit....

Reply:
Right. Have just resolved the merge conflict in light with recent changes to master.

Reply:
ok, @paramsingh  does this look ok?